### PR TITLE
feat(person): add short bio

### PR DESCRIPTION
**Pull request by matthewmayer**
I was working on some fake user profiles recently using Faker, and one thing I felt was missing which is pretty common on social sites is a user bio.
Had a go at adding some patterns for making bios in this draft PR, using a combination of common bio phrases, nouns and emoji
Not fully tested yet

Here's 100 outputs from `faker.person.bio()`
```
gamer, veteran
dreamer, business owner
diary supporter, designer
fava supporter, developer 🔙
clank lover
dealing advocate
uncertainty devotee  😋
zoot-suit lover
ego lover, friend 🐀
animal junkie, environmentalist
damage advocate  🚘
person, founder, creator 🏜️
artist, grad
circumference advocate, author 🟣
baby devotee  👇🏼
spiral junkie  ✉️
model
bonfire devotee
justification advocate, friend
significance enthusiast, philosopher
foodie, coach
scientist, traveler
miss enthusiast, student
philosopher, patriot
vol devotee, designer 🧦
purity devotee, dreamer
patriot, philosopher, patriot
lounge fan
leader
mouser fan, gamer
friend
overexertion fan
skill advocate
patrimony devotee, dreamer
manifestation advocate, coach
pudding lover  🧑🏽‍🎓
product junkie, patriot
designer
gamer, founder, entrepreneur 😴
economics junkie, leader 👎🏿
atmosphere advocate, developer 📐
numismatist junkie
patriot, business owner
engineer, foodie, leader 🍚
veteran, nerd
bulldozer fan, gamer
founder, traveler
exaggeration advocate  🙇🏾‍♀️
reluctance advocate
plate supporter, coach 🇰🇬
leader
dory lover, artist
traveler
mezzanine enthusiast  🐵
disk supporter  🤞🏾
inventor, leader, entrepreneur
doc advocate, artist
traveler, engineer, film lover 🧑🏽‍💻
bribery devotee, geek
lyrics enthusiast, nerd 🌆
nerd, geek
pat devotee, geek
artist
tolerant supporter, entrepreneur
going fan  ⛪
maker lover, author
gamer
nerd
sushi fan
estimate fan  🇲🇭
educator, veteran, author
contrail supporter, streamer 😬
starboard junkie  👩‍❤️‍👨
driving supporter  🇱🇺
veteran, environmentalist
nickname fan, foodie ♠️
timber advocate, scientist
editing junkie
geek, leader
coach
SUV enthusiast, scientist
implementation advocate
environmentalist, artist, philosopher 😢
bugle enthusiast, coach
blogger
leader, educator, person 🐾
complaint fan
friend, veteran
government devotee, creator
cornerstone enthusiast
student, patriot, engineer
vaulting lover, dreamer ✊🏽
typeface junkie, engineer
film lover, traveler, developer 🈵
dependency enthusiast  😜
scientist, educator, traveler
variety junkie
earplug enthusiast, foodie 👆🏻
anise enthusiast, singer 👨🏻‍🤝‍👨🏽
fulfillment advocate  🚄
```

### CODE REVIEW

File: src/modules/person/index.ts [hunk 1]
```diff
@@ -239,6 +239,20 @@ export class PersonModule {
     return this.faker.helpers.objectValue(Sex);
   }
 
+  /**
+   * Returns a random short biography
```
Comment (ST-DDT) [inline src/modules/person/index.ts:244]:
```suggestion
   * Returns a random short biography.
```

**Comment by ejcheng** [comment]
I really like this feature, we should accept it

**Review by Shinigami92 — APPROVED** [review]
I don't have strong opinion with this feature :thinking: :shrug: 

I can assume that later there could also be a long version because bios I know are written in multiple lines
This can be extended with `options: { format: 'short' | 'long' = 'short' }` or anything like that

**Review by ejcheng — APPROVED** [review]

**Comment by ST-DDT** [comment]
1) I'm not sure about the bio supporter and bio parts.
IMO that is just a job title and we could use our job name generation for that.

2) Also we should probably use a custom separator. Or are `bio`s always like this?
`- ` vs `, `

3) Maybe instead of using a complex pattern we could just repeat it (multiple) and join them together.

**Comment by matthewmayer** [comment]
> 1. I'm not sure about the bio supporter and bio parts.
>    IMO that is just a job title and we could use our job name generation for that.
> 2. Also we should probably use a custom separator. Or are `bio`s always like this?
>    `- ` vs `, `
> 3. Maybe instead of using a complex pattern we could just repeat it (multiple) and join them together.

1. My feeling is that social bios are more informal. `faker.person.jobTitle()` gives you something like "Principal Research Producer" which you might display on your LinkedIn, but if you're faking something like Twitter or TikTok, people tend to not put formal job titles, but fuzzier things and more personal things "Dreamer, Influencer, Parent, FOO Lover" etc. 

2/3 I tried to keep the implementation as simple as possible, which means the patterns are a bit more complex (though IMHO still fairly readable). An alternative would be to do more work in code and less in patterns, but i felt that might make future localization harder.

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**